wordpress-mobile/WordPress-FluxC-Android
# Block editor settings endpoint - Removes experimental namespace

## Description
* Aligns the api client with the [removal of the experimental namespace from the block editor settings endpoint](https://github.com/WordPress/gutenberg/pull/33128).
* Adds the site information in the wp-block-editor namespace for WP.com connected sites 
* It also [renames](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2049/commits/d35db4ef45705bc008966e7d81b603541f079f7c) functions and constants in the implementation to clarify that the block editor settings api call escapes the scope of the Global Styles feature.

## To test
Check [Android PR](https://github.com/wordpress-mobile/WordPress-Android/pull/15002)

📓 Note that the Jetpack side is still dependent on some backend changes so this PR is still a draft.
